website/docs/language/data-sources/index.mdx

```diff
@@ -165,8 +165,26 @@ as defined for managed resources, with the same syntax and behavior.
 
 ## Lifecycle Customizations
 
-Data resources do not have any customization settings available
-for their lifecycle. However, the `lifecycle` block is reserved for future versions.
+Data resources support a slightly simpler `lifecycle` block that only includes the `enabled` argument to conditionally enable or disable data sources.
```

Comment:
This sentence assumes that the reader is already familiar with `lifecycle` from `resource` blocks and is expecting it to work similarly, but I don't think we ought to make that assumption, so I suggest the following instead:

```suggestion
Data resources support a `lifecycle` block that includes the [`enabled`](../../language/meta-arguments/enabled.mdx) argument to conditionally enable or disable the data resource.
```

(I think that including a link to the main page about `enabled` should also be sufficient to remove the paragraph at the end of this section which currently links to the `lifecycle` documentation, because there's no longer anything there that's relevant to data resources which isn't also present on the `enabled` page.)